toolchain/lower/file_context.cpp

```diff
@@ -225,6 +225,13 @@ auto FileContext::BuildType(SemIR::NodeId node_id) -> llvm::Type* {
           GetType(array_type.element_type_id),
           sem_ir_->GetArrayBoundValue(array_type.bound_id));
     }
+    case SemIR::ClassType::Kind: {
+      auto object_representation_id =
+          sem_ir_->classes()
+              .Get(node.As<SemIR::ClassType>().class_id)
+              .object_representation_id;
```

Comment:
Stray thought, not specific to this change. If you're writing code like this frequently, you could add to the classes value store (needing a wrapper class) something like:

```
auto Get(Node node) -> Class { return Get(node.As<SemIR::ClassType>().class_id); }
```

Reply:
Mm, something to think about. Sometimes we have a `ClassDeclaration` instead of a `ClassType`, though, so maybe we'd want `Get(Node)` to handle both cases. But then that means adding a branch where we don't have one today.